docs/modules/getting-started/pages/wan.adoc

```diff
@@ -0,0 +1,282 @@
+
+{description}
+
+WAN Replication can work in two modes:
+
+* Active-Passive: An active cluster replicates its maps and/or caches to one or more passive clusters
+for the purpose of maintaining a backup. If the active cluster becomes unavailable, you can redirect the application traffic to the passive cluster.
+* Active-Active: Each cluster replicates maps and/or caches to all other clusters.
+
+See xref:wan:wan.adoc[here] for an overview of the WAN Replication feature.
+
+This tutorial is for the Active-Passive mode, and you will:
+
+. Start an Enterprise cluster as a passive one
+. Start another cluster with WAN Replication configured, this will be the active one
+. Create a map in the active cluster with data in it and see the replication to the passive cluster
+. Modify some map entries in the active cluster and see the modifications in the passive one
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Trial license
+|https://trialrequest.hazelcast.com/[Hazelcast website]
+
+|Docker image for the full Hazelcast distribution
+|xref:get-started-enterprise.adoc[Install Hazelcast Enterprise]
+
+|Docker network with the name `hazelcast-network`
+|Use the `docker network create hazelcast-network` command 
+
+|===
+
+== Step 1. Start the Passive Cluster
+
+In this step you'll start an Enterprise cluster called `london` with a single member. This will be the **passive** cluster
+which means it does not need WAN Replication to be configured. Run the following Docker command on a terminal.
+
+[source,shell,subs="attributes+"]
+----
+docker run \
+    --network hazelcast-network \
+    -e HZ_LICENSEKEY=<your license key> \ <1>
+    -e HZ_CLUSTERNAME=london \
+    -p 5701:5701 hazelcast/hazelcast-enterprise:{full-version}
+----
+<1> Replace with your actual Hazelcast Enterprise license key
+
+The member is up and running now.
+
+[source,shell,subs="attributes+"]
+----
+2021-11-23 11:08:15,055 [ INFO] [main] [c.h.i.c.ClusterService]: [172.18.0.3]:5701 [london] [{full-version}] 
+
+Members {size:1, ver:1} [
+	Member [172.18.0.3]:5701 - bed20746-1505-449b-9f4a-548bcdbe12b8 this
+]
+----
+
+Note the member's IP address and port since you are going to use it while configuring WAN
+replication in the active cluster and connecting Management Center.
+
+== Step 2. Start the Active Cluster
+
+Now you'll start another Enterprise cluster called `tokyo` with a single member. This will be the **active** cluster
+which means it does not need WAN Replication to be configured.
+
+Since the configuration for WAN Replication is a bit more detailed, you are going to use a Hazelcast configuration
+file and pass it to the Docker command, instead of giving all the configuration options in a single command.
+
+. In an existing or new directory, place a hazelcast.yaml (or xml) file with the following content.
```

Comment:
Shouldn't this say to create a directory called `~/config-tokyo`? since this is the one that we mount on the Docker container?